libraries/bdp-access-proxy/bdp-access-api-capture/src/main/java/com/bosch/osmi/bdp/access/mock/generator/MockDataGenerator.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2015.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License Version 2.0 as published by the
+ * Free Software Foundation with classpath exception.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License version 2.0 for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program (please see the COPYING file); if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+package com.bosch.osmi.bdp.access.mock.generator;
+
+import com.bosch.osmi.bdp.access.api.model.BdpEntity;
+import com.bosch.osmi.bdp.access.api.model.User;
+import com.bosch.osmi.bdp.access.impl.BdpApiAccessImpl;
+import com.bosch.osmi.bdp.access.impl.util.Util;
+import com.google.gson.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.Properties;
+
+/**
+ * The MockDataGenerator iterates over all public methods of all api entities and stores the return values in a json
+ * file. It uses the Java Reflection API to inspect the bdp-access-api and google.gson to serialize the result.
+ *
+ * @author muj1be
```

Comment:
Authorship note in this and other classes in this commit is not very telling. Consider using email. Or don't. Up to you.